src/overlays/actors/ovl_Obj_Etcetera/z_obj_etcetera.h

```diff
@@ -3,13 +3,31 @@
 
 #include "global.h"
 
+#define DEKU_FLOWER_TYPE(thisx) (((thisx)->params & 0xFF80) >> 7)
+
+typedef enum {
+    /* 0 */ DEKU_FLOWER_TYPE_PINK,
+    /* 1 */ DEKU_FLOWER_TYPE_PINK_SPAWNED_FROM_MAD_SCRUB,
+    /* 2 */ DEKU_FLOWER_TYPE_GOLD,
+    /* 3 */ DEKU_FLOWER_TYPE_GOLD_SPAWNED_FROM_MAD_SCRUB,
+    /* 4 */ DEKU_FLOWER_TYPE_MAX,
+} DekuFlowerType;
+
 struct ObjEtcetera;
 
 typedef void (*ObjEtceteraActionFunc)(struct ObjEtcetera*, GlobalContext*);
 
 typedef struct ObjEtcetera {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x13C];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x015C */ SkelAnime skelAnime;
+    /* 0x01A0 */ ColliderCylinder collider;
+    /* 0x01EC */ Vec3s jointTable[11];
+    /* 0x022E */ Vec3s morphTable[11];
+    /* 0x0270 */ f32 intenseOscillationScale;
+    /* 0x0274 */ s16 oscillationTimer;
+    /* 0x0276 */ u16 burrowFlag;
+    /* 0x0278 */ s8 objIndex;
+    /* 0x027C */ Gfx* dList;
     /* 0x0280 */ ObjEtceteraActionFunc actionFunc;
 } ObjEtcetera; // size = 0x284
```

Comment:
```suggestion
    /* 0x000 */ DynaPolyActor dyna;
    /* 0x15C */ SkelAnime skelAnime;
    /* 0x1A0 */ ColliderCylinder collider;
    /* 0x1EC */ Vec3s jointTable[11];
    /* 0x22E */ Vec3s morphTable[11];
    /* 0x270 */ f32 intenseOscillationScale;
    /* 0x274 */ s16 oscillationTimer;
    /* 0x276 */ u16 burrowFlag;
    /* 0x278 */ s8 objIndex;
    /* 0x27C */ Gfx* dList;
    /* 0x280 */ ObjEtceteraActionFunc actionFunc;
} ObjEtcetera; // size = 0x284
```
Since the total size of this struct is only 3 digits, only 3 digits are required